### PR TITLE
Setup/Add Mongo/Mongo Express Docker Compose

### PR DESCRIPTION
**Pull request by kishan-aghera**
## What
Added docker compose setup for PostgreSQL database along with pgAdmin.

- [x] I will abide by the [code of conduct](https://github.com/bhavik2936/docker-compose-files/blob/main/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: mongo-mongo-express/compose.yml
Comment (bhavik2936) [inline mongo-mongo-express/compose.yml:1]: We should explicitly declare network shared among both containers with `bridge` driver.

Meanwhile, I was wondering if we have the facility to specify volume for **mongo-express** for persisting data across container restart.

File: mongo-mongo-express/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#### Configuration
+
+Below configuration are already set by default via `mongoExpressCreds.env` file and can be modified by overriding the same.
+
+| Config   | Value | Comment                          |
+| -------- | ----- | -------------------------------- |
+| Port     | 8081  | Exposed at localhost (127.0.0.1) |
+| Username | root  |                                  |
+| Password | root  |                                  |
+| Server   | mongo |                                  |
```
Comment (bhavik2936) [inline mongo-mongo-express/README.md]: We shall avoid enlisting config `server` in readme, as it indicates the viewer that value of `server` is changeable.

File: mongo-mongo-express/mongoCreds.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+MONGO_INITDB_ROOT_USERNAME=root
+MONGO_INITDB_ROOT_PASSWORD=root
+MONGO_INITDB_DATABASE=mongodb
```
Comment (bhavik2936) [inline mongo-mongo-express/mongoCreds.env]: We should skip specifying `MONGO_INITDB_DATABASE` as no entrypoint scripts are mentioned.

Ref: https://hub.docker.com/_/mongo

**Review by bhavik2936 — COMMENTED** [review]
Thanks for your contribution Kishan!

I have couple of suggestions along with concerns regarding volumes & networks (to specify) in compose file; which I have mentioned below.

**Comment by kishan-aghera** [comment]
@bhavik2936  Please check the added commits to find the solutions to the changes you suggested. Thanks.

**Comment by bhavik2936** [comment]
Thanks for the contribution Kishan, Looks Great!

**Pull request merged**